### PR TITLE
Update name to jenkins-helper in parentsJobPipeline

### LINKED ISSUE #2626: Update parentsJobPipeline to reference jenkins-helper
https://github.com/adoptium/aqa-tests/blob/master/buildenv/jenkins/parentsJobPipeline.groovy#L52 should be updated to reference the renamed jenkins-helper at https://github.com/adoptium/jenkins-helper

otherwise hitting this issue:
```
ERROR: No library named openjdk-jenkins-helper found
Finished: FAILURE
```

### PR DESCRIPTION
**Pull request by smlambert**
Fixes #2626

### CODE REVIEW

File: external/common_functions.sh [hunk 1]
```diff
@@ -219,6 +219,21 @@ function set_test_info() {
...
+        environment_variable="MODE=\"java\""
+        debian_packages="git maven"
+        debianslim_packages="${debian_packages}"
+        ubuntu_packages="${debian_packages}"
+        alpine_packages="git maven"
+        centos_packages="git maven"
+        clefos_packages="${centos_packages}"
+        ubi_packages="git maven"
+        ubi_minimal_packages="${ubi_packages}"
+        ;;
```
Comment (llxia) [inline external/common_functions.sh:236]: Are these external test changes intended for this PR?

Comment (smlambert) [reply external/common_functions.sh:236]: crikey, no they are not!

**Comment by llxia** [comment]
I think this PR will affect other Jenkins. We need to update the openj9 and internal Jenkins configuration as well. We should update to use the git URL: https://github.com/adoptium/jenkins-helper and the updated libaray name.

@AdamBrousseau could you please update openj9 Jenkins https://openj9-jenkins.osuosl.org once this PR is merged? I do not have permission. Also, did we update the internal mirror to point new git URL?

**Comment by smlambert** [comment]
This is not urgent to merge, so we can coordinate, as I can point to my branch for the short-term for the configuration at the private server where I am hitting the issue.

**Comment by sophia-guo** [comment]
This PR shouldn't affect other jenkins. The file is only used by a jenkins job to manually trigger external or jck tests. The url can be referenced automatically. Though eventually updating the url is needed.

**Review by sophia-guo — APPROVED** [review]

**Comment by sophia-guo** [comment]
This is a standalone jenkinsfile. Not the one in https://github.com/adoptium/aqa-tests/blob/master/buildenv/jenkins/JenkinsfileBase#L254

**Comment by llxia** [comment]
ah, I see. Should we keep the library name to be the same?

**Review by llxia — APPROVED** [review]
LGTM

**Comment by smlambert** [comment]
Both groovy scripts should eventually get updated to use the real repo name (which is changed to jenkins-helper), but there is no rush to do so necessarily.

**Pull request merged**